schemas/HPXMLDataTypes.xsd

```diff
@@ -1663,6 +1664,13 @@
 			<xs:enumeration value="other"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="AtticWallType">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="gable"/>
+			<xs:enumeration value="knee wall"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
```

Comment:
What other enumerations should I have here? These are walls that go between an attic space and something else.